packages/avatars/demo/stories/AvatarStory.tsx

```diff
@@ -22,11 +22,13 @@ export const AvatarStory: Story<IArgs> = ({ children, type, ...args }) => (
   <Avatar
     {...args}
     backgroundColor={args.backgroundColor || (type === 'image' ? undefined : PALETTE.kale[800])}
+    // for useText - avoids throwing when first inputting text arg passed as an empty string
+    statusLabel={args.statusLabel || args.status}
```

Comment:
This is not preferable, because it masks the dev console warning and expected error. Similar to other components, we want to prove these can happen, as expected, in Storybook. Therefore, these lines can be removed.

At this time, Storybook users are expected to know how to click "Refresh controls" or refresh the URL to get back to a valid state.

Reply:
I disagree - the `Avatar` story is not about testing the effects of the `useText` when providing an empty string. That will happen with usage of the component. For non-developers and new users to Garden, this can be a point of confusion.

Reply:
Per offline conversation, we will remove this in favor of enhancing in the future.